src/main/java/racingcar/domain/CarName.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.domain;
+
+public class CarName {
+
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
```

Comment:
상수와 인스턴스 변수와 구분을 위해 개행을 추가해보면 좋을 것 같습니다.
```suggestion
    private static final int MAX_NAME_LENGTH = 5;

    private final String name;
```